pkg/client/openstack/client.go

```diff
@@ -523,7 +535,10 @@ func (c *client) GetNodes(kluster *kubernikus_v1.Kluster, pool *models.NodePool)
 	err = servers.List(client, opts).EachPage(func(page pagination.Page) (bool, error) {
 		nodes, err = ExtractServers(page)
 		if err != nil {
-			glog.V(5).Infof("Couldn't extract server %v", err)
+			c.Logger.Log(
```

Comment:
Maybe we should log this errors here as the function returns this error to the caller.

Reply:
Yeah, remove that.